data/material-model/entropy-table/opxtable/header_adiabat.txt

```diff
@@ -0,0 +1,7 @@
+# This is an exemplary isentrope for an opx mantle composition, exported
+# from HeFesto, for a surface adiabatic temperature of 1600 K. Temperature
+# column contains entropy for this table.
```

Comment:
The column is called temperature but still contains the entropy? Is that something we can fix?

Reply:
No, the 'ascii data' plugin for adiabatic conditions only knows about temperature. It works with this file, except it stores entropy as temperature. I do not use it by default in the benchmark, but it is important to compare the adiabat ASPECT computes, with the one HeFESTo computes.